ipaserver/install/server/install.py

```diff
@@ -54,6 +54,22 @@
 SYSRESTORE_DIR_PATH = paths.SYSRESTORE
 
 
+def check_password_fips_nssdb_compatible(password):
+    """
+    Check whether the given password can be used for NSSDB setup in FIPS mode
+    """
+    gotnumeric = any(c.isdigit() for c in password[:-1])
```

Comment:
Perhaps we could use a comment that the `:-1` and `1:` is a NSSDB-specific requirement.